src/ui/components/DevTools.js

```diff
@@ -65,16 +64,19 @@ function getUploadingMessage(uploading) {
 }
 
 function getIsAuthorized({ data, error, isAuthenticated }) {
-  if (!features.private) {
+  const test = new URL(window.location.href).searchParams.get("test");
+
+  // Ideally, test recordings should be inserted into Hasura. However, test recordings are currently
+  // not being inserted as a Hasura recordings row, so the GET_RECORDING query will respond with an
+  // empty recordings array. To temporarily work around this for now, we return `true` here so
+  // the test can proceed.
+  if (test) {
```

Comment:
Good call. My bet is that Hasura does not accept a new recording without a user_id. We can probably get around this by having tests use a saved Test account...